### PR TITLE
Dm supports google cloud

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Frank945946 — APPROVED** [review]

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 1]
```diff
@@ -8,6 +8,16 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## November 21, 2023
+
+**General changes**
+
+- TiDB Cloud Data Migration now supports high-speed physical mode in Google Cloud.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-release-notes.md:15]:
```suggestion
- [Data Migration](/tidb-cloud/migrate-from-mysql-using-data-migration.md) supports high-speed physical mode for TiDB clusters deployed on Google Cloud.
```

**Review by lichunzhu — APPROVED** [review]
LGTM

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 2]
```diff
@@ -8,6 +8,16 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## November 21, 2023
+
+**General changes**
+
+- [Data Migration](/tidb-cloud/migrate-from-mysql-using-data-migration.md) supports high-speed physical mode for TiDB clusters deployed on Google Cloud.
+
+    The migration speed of physical mode can reach up to 110 MiB/s, which is 2.4 times faster than logical mode. The improved performance is suitable for quickly migrating a large volume of data to TiDB Cloud. Now you can use physical mode for TiDB clusters deployed on AWS and Google Cloud.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-release-notes.md:17]:
```suggestion
    Now you can use physical mode for TiDB clusters deployed on AWS and Google Cloud. The migration speed of physical mode can reach up to 110 MiB/s, which is 2.4 times faster than logical mode. The improved performance is suitable for quickly migrating large datasets to TiDB Cloud.
```

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 3]
```diff
@@ -214,7 +214,7 @@ You can use **physical mode** or **logical mode** to migrate **existing data**.
 Physical mode has the following limitations:
 
 - When you use physical mode, you cannot create a second migration job or import task for the TiDB cluster before the existing data migration is completed.
-- Physical mode is only available for TiDB clusters deployed on AWS.
+- Physical mode is available for TiDB clusters deployed on AWS and Google Cloud.
```
Comment (Oreoxmt) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:217]: This is not considered a limitation if logical mode is also available on AWS and Google Cloud.

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**